src/modules/edc-demo/components/asset-viewer/asset-viewer.component.html

```diff
@@ -33,13 +33,33 @@
 
     <!-- Paginator -->
     <mat-paginator
+      *ngIf="assetList.isReady"
       hidePageSize="true"
       pageSize="contractDefinitions.length"
-      [length]="assets.length">
+      [length]="assetList.data.filteredAssets.length">
     </mat-paginator>
   </div>
 
-  <edc-demo-asset-list
-    [assets]="assets"
-    (assetClick)="onAssetClick($event)"></edc-demo-asset-list>
+  <div class="flex flex-row uppercase justify-center">
```

Comment:
The uppercase class here is right now making empty messages uppercase.

It should be removed here and empty state / error state should specify such styling.